recipes/pcaone/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Please remove this license file; the `license_file: LICENSE` line in `meta.yaml` will pick up the one packages with your software.

Reply:
done!